src/cmd.rs

```diff
@@ -127,6 +127,9 @@ pub enum SubCommand {
     )]
     ShowVenvBin {},
 
+    #[structopt(name = "tidy", about = "Re-generate a clean lock")]
```

Comment:
What this does is unclear to me. From what I understand you:

- destroy the venv
- create a new venv
- install all deps with the current lockfile (does this reinstall the deps that you removed from setup.py?)
- remove useless packages from the lockfile

If you really need to recreate the venv to implement this, maybe at least document it here :)

Reply:
> install all deps with the current lockfile (does this reinstall the deps that you removed from setup.py?)
Nope, that's the point. It uses the `--constraint` argument when running `pip` so that existing deps do not get updated